core-web/libs/sdk/angular/README.md

```diff
@@ -72,158 +377,143 @@ export const appConfig: ApplicationConfig = {
 };
 ```
 
-This way, we will have access to `DOTCMS_CLIENT_TOKEN` from anywhere in our application.
+#### Client Usage
 
-### Client Usage
-To interact with the client and obtain information from, for example, our pages
+Inject and use the client in your components:
 
 ```typescript
 export class YourComponent {
     private readonly client = inject(DOTCMS_CLIENT_TOKEN);
 
-    this.client.page
-        .get({ ...pageParams })
-        .then((response) => {
-            // Use your response
-        })
-        .catch((e) => {
-            const error: PageError = {
-                message: e.message,
-                status: e.status,
-            };
-            // Use the error response
-        })
+    ngOnInit() {
+        this.client.page
+            .get({ url: '/my-page' })
+            .then(response => {
+                // Handle page data
+            });
+    }
 }
 ```
 
-For more information on how to use the dotCMS Client, you can visit the [documentation](https://www.github.com/dotCMS/core/blob/main/core-web/libs/sdk/client/README.md)
+## FAQ
 
-## DotCMS Page API
+### How do I configure and use the dotCMS Client in my Angular application?
 
-The `DotcmsLayoutComponent` requires a `DotCMSPageAsset` object to be passed in to it. This object represents a dotCMS page and can be fetched using the `@dotcms/client` library.
+The dotCMS Client needs to be properly configured and provided at the application level for all dotCMS components to work correctly.
 
-- [DotCMS Official Angular Example](https://www.github.com/dotCMS/core/tree/main/examples/angular)
-- [`@dotcms/client` documentation](https://www.npmjs.com/package/@dotcms/client)
-- [Page API documentation](https://www.dotcms.com/docs/latest/page-api)
+1. **Setup the Client Configuration**:
 
-## Components
+```typescript
+import { ClientConfig } from '@dotcms/client';
 
-### DotcmsLayoutComponent
+const DOTCMS_CLIENT_CONFIG: ClientConfig = {
```

Comment:
This type is different now, its `DotCMSClientConfig`

Reply:
Also it comes from types

Reply:
Done!